vendor_data.py

```diff
@@ -0,0 +1,156 @@
+#! /usr/bin/python
+
+"""This script is meant to help Translation Project Managers with keeping a clear overview of the vendors participating in a project.
+
+The script contains three classes:
+    1. The main parent class ´VendorData´. This class is meant for a general potential vendor profile, without assigning a status (preferred/back-up).
+       It has 3 class arguments ´ProjectName´, representing the project name and ´SourceLang´, the source language the translators will be translating from and ´CatTools´, the possible Cat Tools the translator can use. 
+       It takes 5 arguments:
+            - ´VendorName´: The first and last name of the vendor.
+            - ´TargLang´: the language the translators will be translating into.
+            - ´WordRate´: the translator's word rate in Euro.
+            - ´VendorMail´: The vendor's email address.
+            -  ´CatTool´: the vendor's preferred cat tool.
+    2. The subclass ´PrefVendor´, for the preferred vendors. They are assigned the status "Preferred".
+    3. The subclass ´BackupVendor´ for the back-up vendors. They are assigned the status "Back-up"
+"""
+
+import argparse
+import re
+import pandas as pd
+import os
+import pyinputplus as pyip
+
+class VendorData:
+    ProjectName = "Toyota MM24" #Class attribute no.1, the project the vendors will be working on
+    SourceLang = "English" #Class attribute no.2, the source language
+    CatTools = ["XTM", "Trados Studio", "MemoQ", "Memsource"] #Class attribute no.3, the list of possible Cat Tools
+    Status = "Potential"
+    
+    def __init__(self, VendorName, TargLang, WordRate, VendorMail = "", CatTool = "XTM"):
+        """ Instantiate
+        Args:
+            "VendorName" (str): First and last name of the vendor
+            "VendorMail" (str): the vendor's email address
+            "TargLang" (str): Language the vendor will be translating to
+            "WordRate" (float): Vendor's word rate for new words in Euro
+            "CatTool" (str): Vendor's preferred CAT-tool
+            """
+        self.VendorName = VendorName
+        self.TargLang = TargLang
+        self.WordRate = WordRate
+        self.VendorMail = VendorMail
+        self.CatTool = CatTool
+        
+        regex_mail = "^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$" #regex used to validate email

```

Comment:
Have you tested this? It might need an r before the string to make it a raw string.